### PR TITLE
feat: add Typography italic type

### PR DESCRIPTION
**Pull request by alwaysloseall**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

#30386

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |     Typography增加斜体字支持      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
更新下snapshot.

File: components/typography/index.en-US.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Basic text writing, including headings, body text, lists, and more.
 | mark | Marked style | boolean | false |  |
 | onClick | Set the handler to handle click event | (event) => void | - |  |
 | strong | Bold style | boolean | false |  |
+| italic | Italic style | boolean | false |  |
```
Comment (xrkffgg) [inline components/typography/index.en-US.md:32]: 加上版本

Comment (alwaysloseall) [reply components/typography/index.en-US.md:32]: 版本号呢

**Comment by alwaysloseall** [comment]
> 更新下snapshot.

已更新

**Review by afc163 — APPROVED** [review]

**Pull request merged**